CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `GenericRequest` model for unsupported request types
 - Methods to convert between `IssuedCurrency` and `IssuedCurrencyAmount`
 - Support for ints and floats in the `IssuedCurrency` and `IssuedCurrencyAmount` models (and ints for `XRP`)
+- Makes `autofill` a publicly exposed method
```

Comment:
```suggestion
- `autofill`, a new public method that populates the `fee`, `sequence`, and `last_ledger_sequence` fields of a transaction, based on the current state retrieved from the server the Client is connected to. It also converts all X-Addresses to classic addresses.
```

- Most users probably weren't aware that there was a private autofill method before
- This matches the grammar of other lines (e.g. "`xrpl.utils.str_to_hex` and `xrpl.utils.hex_to_str` helpers") and reads naturally off of "Added", as in: Added `autofill`, a new public method…